ui/mirage/factories/service.js

```diff
@@ -0,0 +1,34 @@
+import { Factory, faker } from 'ember-cli-mirage';
+import { provide } from '../utils';
+
+export default Factory.extend({
+  name: id => `${faker.hacker.noun().dasherize()}-${id}-service`,
+  portLabel: () => faker.hacker.noun().dasherize(),
```

Comment:
supernit! Do faker nouns ever have spaces (assuming this is what `dasherize` is doing)? I would have thought not, but I might be wrong? At the end of the day this doesn't matter, just I'm assuming `dasherize` might be hung off the `String.prototype` so if nomad ever wants to stop adding things to prototypes you'd have to remove this. I might have completely understood here!

P.S. If I am wrong here, I noticed you don't use `dasherize` on the `tags` below - not sure if thats important.

Reply:
It’s true that if we turn off prototype extensions we’d have to change this, but a find-and-replace would reveal that quickly enough. I tried [the example](https://cdn.rawgit.com/Marak/faker.js/master/examples/browser/index.html#hacker) and it did produce “hard drive”.

I feel like it‘s not important that tags are different… that `provide` abstraction makes it harder because it takes the function rather than its output. I could change it if it were critical but I think it’s safe to ignore.

Reply:
cool sounds good! Thanks for looking